Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,31 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for (int i = 0; i < 2; i++)
+            {
+                Console.WriteLine("Enter person " + (i + 1) + " first name:");
+                string fName = Console.ReadLine();
+
+                Console.WriteLine("Enter " + fName + "'s last name:");
+                string lName = Console.ReadLine();
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s age:");
+                int age = Convert.ToInt32(Console.ReadLine());
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s weight in kg:");
+                float weight = float.Parse(Console.ReadLine());
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s height in cm:");
+                float height = float.Parse(Console.ReadLine());
```

Comment:
In order to get rid of the issue (and because this is much more performant and convenient) I suggest you to use interpolated strings.
Example:
![image](https://user-images.githubusercontent.com/12687948/73128696-8c236280-3fd3-11ea-9f9f-eae5367e7471.png)

Reply:
I don't quite understand what you are trying to teach me here. Is it ok if we table this particular concept until it is covered within the course material?